### ISSUE TITLE
Better detection of deleted projects

### ISSUE THREAD

**Issue by rosswintle** [beyondcode/invoker-test-toolkit#2]
In trying to figure out what's going on with #1 I'm clearly poking around and doing things that I shouldn't do.

This is a stupid edge case thing that almost certainly won't happen in the real world, but...

It's possible to open Invoker, open a project and then delete that project before closing Invoker.

I had done this while trying to diagnose #1 and when I returned to Invoker I'd forgotten which project was open.

In this case clicking both the reload button and the Run All button in the top-right did weird things.

I recorded a very short video here: https://share.getcloudapp.com/P8u5XEZz

In short, when taking actions you probably need to detect the absence of the current project as it may have been deleted of moved,

If you decide that deleting or moving a project that you have open in Invoker is a silly thing to do and that of course that will break stuff then that's fine, I'll understand you not wanting to put effort into fixing something that rarely happens.

Thank you!

**Comment by mpociot** [comment]
Hey Ross,

I'm going to close this issue, as it's really a super niche edge case (which also occurs regardless of this plugin).
I'll see if we can easily fix this in Invoker itself though.

Thanks for reporting this.
